io-uring-test/src/main.rs

```diff
@@ -108,6 +108,7 @@ fn test<S: squeue::EntryMarker, C: cqueue::EntryMarker>(
     tests::net::test_tcp_sendmsg_recvmsg(&mut ring, &test)?;
     tests::net::test_tcp_accept(&mut ring, &test)?;
     tests::net::test_tcp_connect(&mut ring, &test)?;
+    #[cfg(not(feature = "ci"))]
```

Comment:
I'd prefer to skip only buf_group part instead of all tcp_buffer_select tests.

Reply:
Okay. I couldn't tell what you had in mind.